requirements-dev.txt

```diff
@@ -18,3 +18,6 @@ pytest>=6.2.2
 requests
 pytest-cov>=2.10.1
 isort==5.7.0
+
+Django==3.2
+sqlparse==0.4.1
```

Comment:
Why do we need it?

Reply:
Oh, yes, we do not need this here. Probable i planned I would use it, but it was no need to do it. 
Removed.